libs/platform/src/lib/components/button/button.component.scss

```diff
@@ -0,0 +1,6 @@
+.truncate_btn__txt {
```

Comment:
let;s keep the naming syntax like rest of hte fundamentals
`<libray>-<component>-<what>` we use dashes as well. 

`.fdp-btn-truncate__txt `

Reply:
Done